solc/CommandLineInterface.cpp

```diff
@@ -1002,6 +1004,11 @@ General Information)").c_str(),
 			po::value<string>()->value_name("all,bmc,chc,none")->default_value("all"),
 			"Select model checker engine."
 		)
+		(
+			g_strModelCheckerTimeout.c_str(),
+			po::value<unsigned>()->value_name("ms"),
+			"Set model checker timeout per query in milliseconds. 0 means no timeout."
```

Comment:
Well, 0 means forcefully no timeout, while omitting the option means a resource limit. So maybe mentioning these in both the standard json part and here would be nice.

Reply:
Will do